src/UriParse.c

```diff
@@ -480,7 +480,7 @@ static const URI_CHAR * URI_FUNC(ParseIpFuture)(URI_TYPE(ParserState) * state,
 	case _UT('v'):
 	case _UT('V'):
 	*/
-		if (first + 1 >= afterLast) {
+		if (afterLast - first < 2) {
```

Comment:
Should the `first >= afterLast` in line 470 be adjusted to `afterLast - first < 1` for uniform conditions?

Reply:
@TimWolla good question. I added four cases of `afterLast - first < 1` on branch `setters` recently and I see 37 cases of `first >= afterLast` on `master`. Are you in clear favor of either approach? I'm tending towards using `first >= afterLast` everywhere (i.e. adjusting the `setters` branch once more) but the only key reason is "fewer changes". I'd even be okay with this not being consistent to the last bit. What do you think?

Reply:
I don't have a strong opinion either way (that's why I selected “Approve”). I just wanted to make sure to diligently review the changes and note everything that came to mind. “Making fewer changes” is a reasonable choice.

Reply:
@TimWolla thanks for sharing your view! I will merge as-is here then and adjust #249 for consistency right after… :+1: